src/OrbitGl/include/OrbitGl/SelectionData.h

```diff
@@ -21,6 +21,10 @@
 // This is meant to hold the data needed to update the data tabs to a specific selection.
 class SelectionData {
  public:
+  enum SelectionType {
```

Comment:
What about using an `enum class`. This would increase the type safety by not allowing implicition conversion from and to ints.

Reply:
Thanks!